windows/azure-pipelines.yml

```diff
@@ -0,0 +1,142 @@
+
+variables:
+  pytorch.build.version: 1.0
+  pytorch.build.number: 1
+  pytorch.branch: master # override the default of build.version
+  pytorch.repo: pytorch
+
+jobs:
+- job: 'Windows_CPU_Wheels_Build'
+  timeoutInMinutes: 0
+  cancelTimeoutInMinutes: 5
+  variables:
+    PACKAGE: wheels
+    CUDA_VERSION: cpu
+  strategy:
+    maxParallel: 10
+    matrix:
+      CPU_3.5_Wheel:
+        DESIRED_PYTHON: 3.5
+      CPU_3.6_Wheel:
+        DESIRED_PYTHON: 3.6
+      CPU_3.7_Wheel:
+        DESIRED_PYTHON: 3.7
+
+  pool:
+    vmImage: 'vs2017-win2016'
+
+  steps:
+  - task: BatchScript@1
```

Comment:
As written, batch files are invoked using the BatchScript@1 (e.g. line 29) task or the script task (e.g. line 51 and 54). It would be nice to use one format or the other.

Reply:
Since the script task doesn't support multiline scripts, I have to choose the first one. But I don't think it's nice to keep a one-line script in a separate file.

Reply:
Yes, I'll need these variables in the script. So it will be done after that issue is fixed.

Reply:
Another question: Can I set additional environmental variables using BatchScript@1 just like that in the script task?
```yaml
  - task: BatchScript@1
    env:
      VARIABLE: VALUE
```